plugins/statuslines/lualine.nix

```diff
@@ -1,260 +1,281 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.lualine;
+  inherit (lib.nixvim) defaultNullOpts mkSettingsRenamedOptionModules;
+  inherit (lib) types mkOption;
 
+  # TODO: do we really need this
```

Comment:
It is used in three options. I don't see any harm it in, unless you think the options themselves don't need to be declared and instead we'll use the freeform type for their definitions?